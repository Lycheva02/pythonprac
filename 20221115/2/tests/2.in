class B:
    rrr = Num()
    def __init__(self, s):
	    self.num = s

print(B(3).rrr)
k, g = B(12), B(429)
print(k.num, g.num, k.rrr, g.rrr)

k.rrr = 12
g.rrr = (19, 23, 745)
print(k.rrr + g.rrr)