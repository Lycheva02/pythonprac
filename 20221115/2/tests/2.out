0
12 429 0 0
15
